.github/workflows/snakemake-ci.yml

```diff
@@ -28,25 +28,28 @@ jobs:
         with: 
           environment-file: recast_helloworld/environment.yml
 
-      - name: List installed environment dependencies 
-        run: | 
-          micromamba list
-
-      - name: Linting
-        uses: snakemake/snakemake-github-action@v1
-        with:
-          directory: '.test'
-          snakefile: 'workflow/Snakefile'
-
       - name: Testing
         uses: snakemake/snakemake-github-action@v1
         with:
-          directory: '.test'
-          snakefile: 'workflow/Snakefile'
+          directory: 'recast_helloworld'
+          snakefile: 'recast_helloworld/Snakefile'
```

Comment:
In the README it would be helpful to note that this is the equivalent to

```
snakemake --directory recast_helloworld/ --snakefile recast_helloworld/Snakefile --cores 1 --use-conda --conda-cleanup-pkgs cache
```

Reply:
This is now https://github.com/AndriiPovsten/Snakemake-backend-for-RECAST/issues/11, so resolving this comment.